Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,258 @@
+
+let chance1 = """
++---+
+|   |
+O   |
+|
+|
+=====
+"""
+let chance2 = """
+ +---+
+ |   |
+ O   |
+ ||  |
+ |   |
+     |
+ =========
+"""
+let chance3 = """
+ +---+
+ |   |
+ O   |
+|||  |
+ |   |
+     |
+=========
+"""
+let chance4 = """
++---+
+ |   |
+ O   |
+|||  |
+ |   |
+|    |
+=========
+"""
+let chance5 = """
++---+
+ |   |
+ O   |
+|||  |
+ |   |
+| |  |
+=========
+"""
+let chance6 = """
++---+
+ |   |
+ X   |
+|||  |
+ |   |
+| |  |
+=========
+"""
+
+print("Welcome to the game of hangman (countries edition). This game will test the knowledge you have for the countries around the world 🌍.")
+print()
+sleep(1)
+print("You will have a total of 6 chances to guess the word that was randomly selected inorder to beat the game. Once you have used your 6th chance the game is end. So guess your words wisely.")
+print()
+sleep(1)
+print("""
+
+888               888                   888888b.                     🌍🌍
+888               888                   888  "88b                    🌍🌍
+888               888                   888  .88P
+888       .d88b.  888888 .d8888b        8888888K.   .d88b.   .d88b.  888 88888b.
+888      d8P🌍Y8b 888    88K            888  "Y88b d8P🌍Y8b d88P"88b 888 888 "88b
+888      88888888 888    "Y8888b.       888    888 88888888 888🌍888 888 888  888
+888      Y8b.     Y88b.       X88       888   d88P Y8b.     Y88b 888 888 888  888
+88888888  "Y8888   "Y888  88888P'       8888888P"   "Y8888   "Y88888 888 888  888
+                                                                 888
+                                                            Y8b d88P
+                                                             "Y88P"
+""")
+print()
+sleep(1)
+print("If you are ready to take on the challenge type \"ready\" to begin the game.")
+//sleep(1)
+
+let startGame = "ready"
+var userStart = readLine()?.lowercased().trimmingCharacters(in: .whitespaces)
+
+repeat {
+    if userStart == startGame {
+        print()
+        print("Welcome to the game, glhf(good luck have fun)")
+    }
+    else {
+        print()
+        print("Please play the game. Its kinda fun")
+        userStart = readLine()
+    }
+} while userStart != startGame
+
+print()
+//sleep(1)
+
+print("Thank you for starting the game.")
+
+sleep(1)
+
+var userGuessLetter = ""
+var userTries = 0
+var chosenWord = "".lowercased()
+var chosenWord2 = "".lowercased()
+var alphabets = "abcdefghijklmnopqrstuvwxyz"
+
+if let randomWords = hangmanWords.randomElement()?.lowercased() {
+    chosenWord = randomWords
+}
+
+var arrayDashes = [Character]()
+
+for _ in chosenWord {
+    arrayDashes.append("_")
+}
+
+var chosenWordsArray = Array(chosenWord)
+var chosenWordArray2 = Array(chosenWord2)
+
+
+print()
+
+print("Guess the word above by typing your letters.")
+
+var condition = true
+var restartGame = true
+
+func restartHangman () {
+    userGuessLetter = ""
+    userTries = 0
+    chosenWord = "".lowercased()
```

Comment:
This lowercased the display of the word once the user is out of guesses but it does not lowercase it while the user is guessing.